### PR TITLE
cosmetic changes

### PR DESCRIPTION
**Pull request by brotzeit**

### CODE REVIEW

File: rust_src/src/marker.rs [hunk 1]
```diff
@@ -30,39 +30,39 @@ impl LispMarkerRef {
         unsafe { mem::transmute(self.as_ptr()) }
     }
```
Comment (shaleh) [inline rust_src/src/marker.rs]: I am not opposed to moving to `isize`. However, we should be consistent. Does this mean we also drop the `c_int` usage as well?

File: rust_src/src/symbols.rs [hunk 2]
```diff
@@ -23,6 +23,10 @@ use crate::{
 pub type LispSymbolRef = ExternalPtr<Lisp_Symbol>;
 
 impl LispSymbolRef {
```
Comment (shaleh) [inline rust_src/src/symbols.rs]: Does it make more sense to drop this style and only use `From` traits?

Comment (brotzeit) [reply rust_src/src/symbols.rs]: You mean `as_lisp_obj` ?

File: rust_src/src/marker.rs (hunk 1 above)
Comment (brotzeit) [reply rust_src/src/marker.rs]: I thought we only use C types when rust functions are used by C functions.

File: rust_src/src/symbols.rs (hunk 2 above)
Comment (shaleh) [reply rust_src/src/symbols.rs]: Yes.

Comment (brotzeit) [reply rust_src/src/symbols.rs]: Makes sense to me. This method has always looked a little strange.

**Pull request merged**